app/controllers/projects_controller.rb

```diff
@@ -67,18 +69,15 @@ def create
   def show
     project
     @departments = project.departments.join(", ")
-    @project_metadata = project.metadata
+    @project_metadata = project.metadata_model
 
-    sponsor_uid = @project_metadata[:data_sponsor]
-    @data_sponsor = User.find_by(uid: sponsor_uid)
+    @data_sponsor = User.find_by(uid: @project_metadata.data_sponsor)
```

Comment:
I do not think this line is needed twice
```suggestion
    @data_sponsor = User.find_by(uid: @project_metadata.data_sponsor)
```